spark-jobs/src/test/scala/za/co/absa/enceladus/conformance/interpreter/fixtures/StreamingFixture.scala

```diff
@@ -0,0 +1,65 @@
+/*
+ * Copyright 2018 ABSA Group Limited
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package za.co.absa.enceladus.conformance.interpreter.fixtures
+
+import org.apache.spark.sql.{DataFrame, Row}
+import org.apache.spark.sql.catalyst.encoders.RowEncoder
+import org.apache.spark.sql.execution.streaming.MemoryStream
+import org.apache.spark.sql.streaming.Trigger
+import org.scalatest.FunSuite
+import org.scalatest.mockito.MockitoSugar
+import za.co.absa.enceladus.conformance.{ConfCmdConfig, HyperConformance}
+import za.co.absa.enceladus.conformance.interpreter.FeatureSwitches
+import za.co.absa.enceladus.conformance.streaming.{InfoDateFactory, InfoDateLiteralFactory}
+import za.co.absa.enceladus.dao.MenasDAO
+import za.co.absa.enceladus.model.Dataset
+import za.co.absa.enceladus.utils.testUtils.SparkTestBase
+
+trait StreamingFixture extends FunSuite with SparkTestBase with MockitoSugar {
+  implicit val menasBaseUrls: List[String] = List()
+  implicit val cmd: ConfCmdConfig = ConfCmdConfig.apply(reportVersion = Some(1))
+  implicit val infoDateFactory: InfoDateFactory = new InfoDateLiteralFactory("2020-05-23")
+  private val interval = 10000
```

Comment:
How this interval was chosen?

Reply:
Mostly I lowered the interval as long as it was possible to reproduce the bug. Since on the test where the optimizer workaround is enabled, it works to write the results during this interval, but on the other one it doesn't. From testing with several values I noticed not enabling this interval would keep the test stuck, so adding the timeout made sense.